src/components/PostCard/PostCard.module.scss

```diff
@@ -0,0 +1,20 @@
+* {
+    box-sizing: border-box;
+}
```

Comment:
This should be placed in index.scss to avoid repeating the code in every component
This applies to all of your .scss files